src/FormModel.php

```diff
@@ -150,10 +150,18 @@ public function hasErrors(?string $attribute = null): bool
         return $attribute === null ? !empty($this->attributesErrors) : isset($this->attributesErrors[$attribute]);
     }
 
+    /**
+     * @param array $data
+     * @param string|null $formName
+     * @return bool
+     */
     public function load(array $data, $formName = null): bool
```

Comment:
Can't we adjust signature itself and remove phpdoc here?

```suggestion
    public function load(array $data, ?string $formName = null): bool
```